README.md

```diff
@@ -0,0 +1,116 @@
+
+**School Library** is a web application designed to facilitate the management and organization of a school library's resources. It provides a digital platform where students, teachers, and librarians can access, search and borrow books as well as perform various library related tasks.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<summary>Back-end</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+  </ul>
+
+## 🚀 Project Demo <a name="project-demo"><a>
+`COMING SOON`
+
+### Key Features <a name="key-features"></a>
+
+- **Save records of who borrowed a given book and when**
+- **Add new books**
+- **Add a new student or teacher**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- At first Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/edubew/School-library.git
+```
+
+- To use this repo to your local computer make sure you have installed [Ruby](https://www.ruby-lang.org/)
+- To install necessary require packages open your favorite cd to this project then try the below command
+```
+gem install
+```
+- To run this project on your local machine `Enter` the below command
+```
+ruby student.rb
+```
+or
+```
+ruby teacher.rb
+```
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Winnie Edube**
+
+- GitHub: [@edubew](https://github.com/edubew)
+- Twitter: [@edube_winne](https://twitter.com/edube_winne)
+- LinkedIn: [Winfred Edube](https://www.linkedin.com/in/winfred-edube/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add a feature for adding book reviews and ratings**
+- [ ] **Add a feature for check-outs and returns, whereby students be able to track due dates and receive reminders for returning borrowed books**
+- [ ] **Add a catalog search for books by title, author, subject or keywords**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/edubew/School-library/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+A heartfelt thank you to Microverse for their invaluable support, guidance, and the incredible opportunity they have provided us. Without them, this project would not have been possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/edubew/School-library/blob/dev/LICENSE) licensed.
```

Comment:
- [ ] Please kindly fix the MIT link, your link is broken and when I click on it I got an error. Kindly fix that 🙏🏾